example/webpack.config.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const createExpoWebpackConfigAsync = require("@expo/webpack-config");
+const fs = require("fs");
+const path = require("path");
+
+const node_modules = path.resolve(__dirname, "..", "node_modules");
+const packages = path.resolve(__dirname, "..", "packages");
+
+module.exports = async function (env, argv) {
+  const config = await createExpoWebpackConfigAsync(env, argv);
+
+  config.context = path.resolve(__dirname, "..");
+
+  config.module.rules.push({
+    test: /\.(js|ts|tsx)$/,
+    include: /(packages|example)\/.+/,
+    exclude: /node_modules/,
+    use: "babel-loader",
+  });
+
+  Object.assign(config.resolve.alias, {
```

Comment:
I think this will encounter the same issue Dripsy had; will need to add

```ts
config.resolve.alias = Object.assign(config.resolve.alias, {
```

Reply:
I think the issue with dripsy was because of the RNW/RN versions not aligning with the other example package. I can add it for save measure though.